src/libstore/nar-info.cc

```diff
@@ -41,6 +41,8 @@ NarInfo::NarInfo(const Store & store, const std::string & s, const std::string &
             compression = value;
         else if (name == "FileHash")
             fileHash = parseHashField(value);
+        else if (name == "IPFSHash")
```

Comment:
Should we call this `IpfsNarHash` or something for future comparability with computing Nars on the fly from some nicer format? Or am I jumping the gun for a file called `nar-info.cc` after all :).